docs/.vuepress/components/SearchBarDoc.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <StackLayout>
+    <SearchBar 
+      :text="searchPhrase" 
+      :hint="hint" 
+      :textFieldBackgroundColor="background"
+      :textFieldHintColor="hintColor"
+      @submit="onSubmit" 
+      @textChange="onTextChange($event)"
+      @clear="onClear"
+      @input="searchPhrase = $event"
+      id="searchbar__docs"
+      />
+    <div class="flex m-t-10">
+      <Label text="Hint:" class="label"></Label>
+      <input v-model="hint" class="form-input" id="searchbar__hint__input">
```

Comment:
Could you convert input to TextField component?

Reply:
done